tools/modules/EnsEMBL/Web/Component/Tools/VEP/Results.pm

```diff
@@ -1265,6 +1251,18 @@ sub get_items_in_list {
       elsif ($item =~ /^(PDB-ENSP_mappings:)((.+)\.\w)$/i) {
         $item_url = "$1&nbsp;".$hub->get_ExtURL_link($2, 'PDB', $3);
       }
+      elsif ($item =~ /^AFDB-ENSP_mappings:(.+)$/i) {
+        # The alphafold ids we store in Ensembl databases (e.g. AF-P63151-F1.A)
+        # are fake ids, which consist of the real alphafold id followed by a dot and a chain name.
+        # Meanwhile, the alphafold site uses Uniprot ids as accession ids.
+        # A Uniprot id is the middle part of an alphafold id (separated by hyphens).
+        # Hopefully, things will improve in the future.
+        my $our_alphafold_id = $1;
+        my ( $actual_alphafold_id ) = $our_alphafold_id =~ /(.+)\./; # without the dot-chain
+        my ( $uniprot_id ) = $actual_alphafold_id =~ /-(.+)-/; # the middle part of an alphafold id
+
+        $item_url = "AFDB-ENSP_mappings:" . "&nbsp" . $hub->get_ExtURL_link($actual_alphafold_id, 'ALPHAFOLD', $uniprot_id);
```

Comment:
i am confused why you need to pass `$actual_alphafold_id` in this function. Do we not need only the uniprot_id?

Example link:
https://alphafold.ebi.ac.uk/entry/P63151

Reply:
`$actual_alphafold_id` is being used as text for the link (the part that goes in between the `<a>` tags). See the implementation of  `$hub->get_ExtURL_link` function.